src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -20,6 +20,13 @@
 					class="tw-relative"
 					@click="showLoanDetails"
 				>
+					<loan-bookmark-exp
+						v-if="!isVisitor"
+						:loan-id="loanId"
+						class="tw-absolute tw-right-1.5 tw-z-2"
+						style="top: -6px;"
```

Comment:
To match the mocks, you'll also want: `right: 8px`

Reply:
I think `tw-right-2` is the same... thanks!

Reply:
Looks like is 16px, so you'll want `tw-right-1`

Reply:
Looks like diff from here: https://tailwindcss.com/docs/top-right-bottom-left

Yeah, we can use `tw-right-1` 👍

Reply:
Those types of settings are based on our custom configs in `kv-components`